modules/redpanda/redpanda_test.go

```diff
@@ -509,19 +514,21 @@ func TestRedpandaWithTLS(t *testing.T) {
 }
 
 func TestRedpandaWithTLSAndSASL(t *testing.T) {
-	tmp := t.TempDir()
+	ctx := context.Background()
 
+	containerHostAddress, err := detectContainerHostAddress(testImage, ctx)
+	require.NoError(t, err, "failed to detect container host address")
+
+	tmp := t.TempDir()
 	cert := tlscert.SelfSignedFromRequest(tlscert.Request{
```

Comment:
fyi: there's now a new method SelfSignedFromRequestE which returns an error, which is more idiomatic.

No need to change but wanted to make you aware.

Reply:
Done